forte/mrdsrg-spin-adapted/sadsrg.cc

```diff
@@ -417,6 +420,9 @@ std::shared_ptr<ActiveSpaceIntegrals> SADSRG::compute_Heff_actv() {
     auto fci_ints =
         std::make_shared<ActiveSpaceIntegrals>(ints_, actv_mos_, actv_mos_sym_, core_mos_);
     fci_ints->set_scalar_energy(Edsrg - Enuc_ - Efrzc_);
+    if (foptions_->get_bool("EMBEDDING_ALIGN_SCALAR") == true) {
```

Comment:
Maybe add a comment?
```suggestion
    if (foptions_->get_bool("EMBEDDING_ALIGN_SCALAR")) {
```